packages/static-wado-creator/lib/operation/ThumbnailService.js

```diff
@@ -59,7 +65,7 @@ class ThumbnailService {
   constructor() {
     this.framesThumbnailObj = [];
     this.favoriteThumbnailObj = {};
-    this.thumbFileName = "thumbnail.jpeg";
```

Comment:
.jpeg doesn't get served by the yarn start service in OHIF, so name it canonically.